README.md

```diff
@@ -108,9 +107,54 @@ print(response.text)
 # Interpret your results using the API - where is the space station right now ?
 ```
 
+### Finding the number of people in space
```

Comment:
Current headings aren't consistent in style. Let's change this to just say `Number of people in space` so that it matches the style of `Current location of International Space Station`